tests/installation/installation_overview.pm

```diff
@@ -69,6 +69,12 @@ sub run {
         $self->deal_with_dependency_issues;
         assert_screen "inst-xen-pattern" if get_var('XEN');
         ensure_ssh_unblocked;
+        # Fail here if we have not graphical.target when we expect it
+        if (get_var('DESKTOP') && !check_var('DESKTOP', 'textmode')) {
```

Comment:
Why cannot we do this check for textmode? As it will be same issue if we got target set to graphical, but expected textmode in the end.

Reply:
Right, let me change this.